.github/workflows/main.yml

```diff
@@ -37,6 +37,9 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run test container
       run: make test-container "CEPH_VERSION=${{ matrix.ceph_version }}" "RESULTS_DIR=$PWD/_results"
+    - name: Run ptrguard tests
+      if: "matrix.ceph_version == 'octopus'"
+      run: make test-container "BUILD_TAGS=with_ptrguard" "CEPH_VERSION=${{ matrix.ceph_version }}" "RESULTS_DIR=$PWD/_results"
```

Comment:
Instead of passing in build tags and then modifying them, I think we should have a dedicated var for ptrguard. USE_PTRGUARD=true or something along those lines.

Reply:
But then we have to do that again and again. Is it not more practical to have this generic solution?

Reply:
I really hope we don't add lots and lots of build tags. If it acts as a small speed bump to prevent tag proliferation, all the better. ;-)

Reply:
Ok, fair point.

Reply:
Done